packages/genesys-spark-components/src/components/beta/gux-rich-text-editor/gux-rich-text-editor.scss

```diff
@@ -1,14 +1,16 @@
 :host {
   display: block;
+  inline-size: 100%;
+  block-size: 100%;
 }
 
 .gux-rich-text-editor-container {
   box-sizing: border-box;
   display: flex;
   flex-direction: column;
   gap: var(--gse-ui-rte-innerContainer-gap);
-  inline-size: 100%;
-  max-inline-size: 100%;
+  inline-size: inherit;
+  max-inline-size: inherit;
```

Comment:
I don't think this is right. If the host is set to `50%` then the container will be `50%` of `50%`.

I thought this change was going to be 

```diff
inline-size: 100%;
max-inline-size: 100%;
+  block-size: 100%;
+  max-block-size: 100%;
```
and no changes on the host.

Reply:
Will look into it